File: Assets/VirtualCity/ProtoDefine/RspConnectOtherServerMessage.cs
using System.Collections.Generic;
using ProtoBuf;
namespace ProtoDefine {
[ProtoContract]


public class RspConnectOtherServerMessage {
[ProtoMember(1)]
    public int code;
[ProtoMember(2)]
    public string serverName;


    public RspConnectOtherServerMessage() {
    }


    public int getCode() {
        return code;
    }

    public void setCode(int code) {
        this.code = code;
    }

    public RspConnectOtherServerMessage(int code, string serverName) {
        this.code = code;
        this.serverName = serverName;
    }

    public string getServerName() {
        return serverName;
    }

    public void setServerName(string serverName) {
        this.serverName = serverName;
    }
}
}

File: Assets/VirtualCity/ProtoDefine/Java/system/RspConnectOtherServerMessage.java
package com.kingston.jforgame.server.game.system.message;

import com.baidu.bjf.remoting.protobuf.annotation.Protobuf;
import com.kingston.jforgame.server.game.Modules;
import com.kingston.jforgame.server.game.system.SystemDataPool;
import com.kingston.jforgame.socket.annotation.MessageMeta;
import com.kingston.jforgame.socket.message.Message;

@MessageMeta(module = Modules.BASE,cmd = SystemDataPool.RSP_CONNECT)
public class RspConnectOtherServerMessage extends Message {
    @Protobuf(order = 1)
    private int code;
    @Protobuf(order = 2)
    private String serverName;


    public RspConnectOtherServerMessage() {
    }


    public int getCode() {
        return code;
    }

    public void setCode(int code) {
        this.code = code;
    }

    public RspConnectOtherServerMessage(int code, String serverName) {
        this.code = code;
        this.serverName = serverName;
    }

    public String getServerName() {
        return serverName;
    }

    public void setServerName(String serverName) {
        this.serverName = serverName;
    }
}
